packages/Chem/src/widgets/properties.ts

```diff
@@ -56,6 +85,70 @@ export function getMoleculeCharge(mol: OCL.Molecule): number {
   return moleculeCharge;
 }
 
+export async function statsWidget(molCol: DG.Column<string>): Promise<DG.Widget> {
+  const host = ui.div();
+  host.style.marginLeft = '-25px';
```

Comment:
can we put it into CSS?

Reply:
Actually, I see no reason for putting only margin left to css. To be discussed tomorrow.